drivers/ovsdriver.go

```diff
@@ -57,14 +61,38 @@ type OvsDriverConfig struct {
 	}
 }
 
+type OvsDriverOperState struct {
+	core.CommonState
+	// used to allocate port names. XXX: should it be user controlled?
+	CurrPortNum int `json:"currPortNum"`
```

Comment:
moving CurrPortNum to Driver's global Oper State is ok. Is it more cosmetic? Because I am still not able to find out how an ep is given back the same port-number, etc. because of this change. EP's restore should come with various port numbers that were needed for those EPs from the ep-oper state.